tests/test_metricsController.py

```diff
@@ -35,15 +35,15 @@
 
 
 def _metricName():
-    return "misc_tasks.FancyMetric"
+    return "FancyMetric"
```

Comment:
This fix is incorrect, because this function is called in places where an unqualified name is invalid, e.g., https://github.com/lsst/verify/blob/d6d5d0ab50cd166cd1f9cbe63050e6342fa9cdb9/tests/test_metricsController.py#L268-L269

However, this file will be going away as soon as I merge DM-33256, so maybe it's not worth worrying about. 🙂

Reply:
#103 merged, so problem solved.